test-files/ui/features/step_definitions/custom_steps.rb

```diff
@@ -334,7 +334,7 @@
 end
 
 When(/^I fill "([^"]*)" with "([^"]*)"$/) do |el, value|
-  find('#' + el).set(value)
+  find(el).set(value)
```

Comment:
This change caused the test to break.  It's passing an id without the hash, expecting the custom step will add it.  If you really want to change the custom step, add the hash to the line below, but there may be more uses of it that need fixin'.  Might be better to fix the uses of "I fill """ in the new test above.

>Unable to find css "horizontal_cookie_cutter_alpha_shape_buffer" (Capybara::ElementNotFound)
./features/step_definitions/custom_steps.rb:337:in `/^I fill "([^"]*)" with "([^"]*)"$/'
features/advanced_options.feature:108:in `Then I fill "horizontal_cookie_cutter_alpha_shape_buffer" with "5"'